docs/openapi-backend.yaml

```diff
@@ -0,0 +1,195 @@
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/QtspClauses"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        default:
+          $ref: "#/components/responses/Unexpected"
+  /signatures:
+    post:
+      operationId: createSignature
+      summary: Create a Signature from SignatureRequest
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateSignatureBody"
+      responses:
+        "201":
+          description: Signature created
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+components:
+  securitySchemes:
+    Bearer:
+      type: apiKey
+      name: Authorization
+      in: header
+  responses:
+    NotFound:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/NotFound"
+    BadRequest:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/BadRequest"
+    Forbidden:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/Forbidden"
+    Unexpected:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/Unexpected"
+  schemas:
+    SignatureRequestDetailView:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/schemas/SignatureRequestDetailView"
+
+    SignatureRequestId:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/schemas/SignatureRequestId"
+
+    QtspClauses:
+      type: object
+      properties:
+        clauses:
+          type: array
+          items:
+            $ref: "#/components/schemas/QtspClause"
+          minItems: 1
+          uniqueItems: true
+        mrcDocumentUrl:
+          type: string
+          format: uri
+        privacyDocumentUrl:
+          type: string
+          format: uri
+        tosDocumentUrl:
+          type: string
+          format: uri
+      required:
+        - text
+        - mrcUrl
+
+    QtspClause:
+      type: object
+      properties:
+        text:
+          type: string
+      required:
+        - text
+
+    PublicKeyDigest:
+      type: string
+      format: base64url
+      description: Hash of PEM public key encoded in base64
+
+    ClausesSignature:
+      type: object
+      properties:
+        signatureFieldId:
+          type: string
+        accepted:
+          type: boolean
+
+    DocumentSignature:
```

Comment:
shouldn't have a `status` properties?

Reply:
this will be changed, but for now let's leave it that way!